human/path.go

```diff
@@ -0,0 +1,32 @@
+package human
+
+import (
+	"bytes"
+	"os"
+	"os/user"
+	"path/filepath"
+)
+
+// Path represents a path on the file system.
+//
+// The type interprets the special prefix "~/" as representing the home
+// directory of the user that the program is running as.
+type Path string
+
+func (p *Path) UnmarshalText(b []byte) error {
+	switch {
+	case bytes.HasPrefix(b, []byte{'~', filepath.Separator}):
```

Comment:
Doesn't `filepath.Separator`/`/` point to the root, rather than the home directory?

Reply:
Yes, my intention here is to have this basically `[]byte("~/")`, but filepath.Separator is a byte (declared as `'/'` or `'\\'`) so I'm constructing the a byte slice with two elements being `~` and the separator.

Did I make a mistake?